Documentation/Contributing/UnitTests.md

```diff
@@ -19,7 +19,7 @@ There's two types of tests that can be added for new code
 ## Play mode tests
 
 Play mode tests will be executed in Unity's play mode and should be added into MixedRealityToolkit.Tests > PlaymodeTests. 
-To create a new test the following template can be used: (todo: Setup() and teardown should be moved into a base class of mrtk tests)
+To create a new test the following template with the can be used:
```

Comment:
Nit: with the ?

Reply:
xD i have no idea how that got in there :D